docs/user_guide/libraries.rst

```diff
@@ -40,22 +44,22 @@ Similarly to :ref:`PDKs<PDKs>`, library modules must implement the following fun
 
    * - Function
      - Description
-     - Arg
+     - Args
      - Returns
      - Used by
      - Required
 
    * - **setup**
      - Library setup function
-     - chip
-     - chip
+     - chip, lib_name
+     - lib
      - :meth:`.load_lib()`
      - yes
 
    * - **make_docs**
      - Doc generator
      - None
-     - chip
+     - lib
      - sphinx
      - yes
```

Comment:
No

Reply:
The make_docs methods return the results of the lib modules' `setup(chip)` functions, which return a Library - not a Chip.

Reply:
The no, was to 'yes', this function is not required any more

Reply:
Ah, got it - thanks for pointing that out.